### PR TITLE
Salty marines updated and added some desires

### PR DESCRIPTION
**Pull request by Krarilotus**

### CODE REVIEW

**Review by Krarilotus — COMMENTED** [review]
looks good, but is specific to these build choices

File: StarCraft2Bot/Builds/BuildServices/ExpandForever.cs
Comment (Aceto1) [inline StarCraft2Bot/Builds/BuildServices/ExpandForever.cs:1]: This is not our custom structure, should this still be included?

File: StarCraft2Bot/StarCraft2Bot.csproj
Comment (Aceto1) [inline StarCraft2Bot/StarCraft2Bot.csproj:1]: This needs to be reverted since the file is deleted

File: StarCraft2Bot/Builds/BansheesAndMarines.cs
Comment (Aceto1) [inline StarCraft2Bot/Builds/BansheesAndMarines.cs:1]: This is not our custom structure, should this still be included?

**Pull request closed without merging**